src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const preparedQuery = query.trim().toLowerCase();
```

Comment:
Let's move this `preparedQuery` out of the filter method.😃
We need it to avoid creating a new variable at each iteration.